2.adoc

```diff
@@ -72,5 +72,94 @@ Optional description
 . Minus A
 . Minus B
 
+=== LibreOffice Writer
+LibreOffice Writer je multiplatformový textový preprocesor z open-source kancelárského balíka LibreOffice.  
```

Comment:
Pridal bych poznamku proc jsme si ho vybrali. Ze si uvedomujeme, ze je to mimo platformu, ale vzhledem k tomu, ze jsme delali celkem 7 resersi, tak jsme zvolili i ciste kancelarskou aplikaci, ktera ma dlouhej vyvoj a je to program, kterej je specializovanej na psani obsahove orientovanejch dokumentu. (nemusi tam byt presne co pisu, jen co me napadlo, abys to nemusel cucat z prstu).